### ISSUE TITLE
Create connector for PubSub

### ISSUE THREAD

**Issue by panys9** [open-metadata/OpenMetadata#13643]
**Is your feature request related to a problem? Please describe.**
There are no connector for ingesting metadata from GCP PubSub

**Describe the solution you'd like**
Create a connector for GCP PubSub as a messaging entity

**Describe alternatives you've considered**
Can create a custom connector but needs to be maintained and is harder to configure by the user.

**Additional context**

**Comment by heavylemon1** [comment]
Hello @panys9 @pmbrull ! :wave: 

New to the project here, I saw this issue and wanted to know if I can pick it up and start working on it.
I have some experience with Data Engineering concepts like ingestions, and some experience working with databases which might help me on this issue.

If that's not the case, that's completely understandable by my side.
Thanks!

**Comment by coreycasmedes** [comment]
I've started working on this anyone is interested
https://github.com/coreycasmedes/OpenMetadata/tree/ISSUE-13643

**Comment by haithem-souala** [comment]
+1

**Comment by SimonDegeorge** [comment]
It would be great to have this feature !

**Comment by pmbrull** [comment]
hi @coreycasmedes are you still working on it?

**Comment by coreycasmedes** [comment]
> hi @coreycasmedes are you still working on it?

Yes. I was busy for the last month but now I'm actively working on it again

**Comment by koushiksk** [comment]
Is this issue still open, if it is I would like to contribute. Can you assign it to me

**Comment by rsoaresp** [comment]
Hi @koushiksk! We are interested in this feature too! Are you still actively involved on implementing this?

**Comment by koushiksk** [comment]
Hi @rsoaresp yes, I will raise the PR by this weekend and let you know.

**Comment by martaradzi** [comment]
Is the new connector scheduled to be released already? Do we know when it would become available?
